src/main/java/crawlercommons/filters/basic/BasicURLNormalizer.java

```diff
@@ -255,6 +262,162 @@ public String filter(String urlString) {
         return urlString;
     }
 
+    private static URL parseUrl(String urlString) {
```

Comment:
I would really appreciate here a JavaDoc explaining what this method is doing (Even though it is private)

Reply:
done.